desktop/src/main/java/bisq/desktop/main/offer/MutableOfferViewModel.java

```diff
@@ -1208,4 +1216,35 @@ private void stopTimeoutTimer() {
     private BSFormatter getFormatterForMakerFee() {
         return dataModel.isCurrencyForMakerFeeBtc() ? btcFormatter : bsqFormatter;
     }
+
```

Comment:
The code works as expected atm, but I think this part of the could should be moved into the MutableOfferViewDataModel.

Reply:
I'm not sure what part of the code should I move there. It seems like I can move just a few lines of the actual math calculation, while I'll need to keep the rest. Is that what you wanted me to move?

Reply:
I've to look at the code in more detail. I'll give it a quick try if I'm able to move it without triggering undesired updates.

Reply:
On second look, yes it is not really feasible to move everything into the data model. Please move only the calculation part as you suggested to the data model. There is already a block of calculate... methods there. Besides that and the naming of the update method ACK